bridge/opencensus/trace_test.go

```diff
@@ -44,3 +45,105 @@ func TestOCSpanContextToOTel(t *testing.T) {
 	got := OCSpanContextToOTel(input)
 	assert.Equal(t, want, got)
 }
+
+func TestOTelSpanContextToOC(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    oteltrace.SpanContext
+		expected octrace.SpanContext
+	}{
+		{
+			name:  "empty span context",
+			input: oteltrace.SpanContext{},
+			expected: octrace.SpanContext{
+				TraceID:      [16]byte{},
+				SpanID:       [8]byte{},
+				TraceOptions: octrace.TraceOptions(0),
+			},
+		},
+		{
+			name: "sampled span context",
+			input: oteltrace.NewSpanContext(oteltrace.SpanContextConfig{
+				TraceID:    oteltrace.TraceID{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+				SpanID:     oteltrace.SpanID{1, 2, 3, 4, 5, 6, 7, 8},
+				TraceFlags: oteltrace.FlagsSampled,
+			}),
+			expected: octrace.SpanContext{
+				TraceID:      [16]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+				SpanID:       [8]byte{1, 2, 3, 4, 5, 6, 7, 8},
+				TraceOptions: octrace.TraceOptions(1),
+			},
+		},
+		{
+			name: "not sampled span context",
+			input: oteltrace.NewSpanContext(oteltrace.SpanContextConfig{
+				TraceID: oteltrace.TraceID{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+				SpanID:  oteltrace.SpanID{1, 2, 3, 4, 5, 6, 7, 8},
+			}),
+			expected: octrace.SpanContext{
+				TraceID:      [16]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+				SpanID:       [8]byte{1, 2, 3, 4, 5, 6, 7, 8},
+				TraceOptions: octrace.TraceOptions(0),
+			},
+		},
+		{
+			name: "span context with tracestate",
+			input: func() oteltrace.SpanContext {
+				ts := oteltrace.TraceState{}
+				ts, _ = ts.Insert("key1", "value1")
+				ts, _ = ts.Insert("key2", "value2")
+				return oteltrace.NewSpanContext(oteltrace.SpanContextConfig{
+					TraceID:    oteltrace.TraceID{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+					SpanID:     oteltrace.SpanID{1, 2, 3, 4, 5, 6, 7, 8},
+					TraceState: ts,
+				})
+			}(),
+			expected: octrace.SpanContext{
+				TraceID:      [16]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16},
+				SpanID:       [8]byte{1, 2, 3, 4, 5, 6, 7, 8},
+				TraceOptions: octrace.TraceOptions(0),
+				// Tracestate will be set by the conversion function
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := OTelSpanContextToOC(tt.input)
+
+			// Verify TraceID
```

Comment:
Those comments are too verbose. No need to describe what the following line does.

```suggestion
```